library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -47,15 +47,16 @@ internal class TransactionPayloadFragment :
     private var backgroundSpanColor: Int = Color.YELLOW
     private var foregroundSpanColor: Int = Color.RED
 
-    private var type: Int = 0
+    private val payloadType: PayloadType by lazy {
```

Comment:
```suggestion
    private val payloadType: PayloadType by lazy(LazyThreadSafetyMode.NONE) {
```
Given that the `payloadType` is accessed only by the main thread.

Reply:
Interesting. Didn't know about this thing. Thanks.